templates/records.html

```diff
@@ -2,16 +2,17 @@
 <html>
 
 <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>iReporter</title>
-    <link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="stylesheet" type="text/css" media="screen" href="css/style.css" />
-    <script src="main.js"></script>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="text/html" charset="utf-8" />
+        <link rel="stylesheet" type="text/css" media="screen" href="../css/style.css" />
+        <script type="text/javascript" src="../js/functions.js"></script>
```

Comment:
You might want to put your .js files at the botom of your body. Sometimes you js getElement will fail to find ids.